### ISSUE TITLE
index.php

### ISSUE THREAD

**Issue by vidon123** [vidon123/Web#1]
<?php session_start(); ?>

<!DOCTYPE html>
<html>
<head>
	<meta name="viewport" content="width=device-width" />
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<link rel="shortcut icon" type="image/png" href="/favicon.png"/>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
	<script src="js/fleche.js"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
	<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.4/jquery.min.js"></script>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.0.2/anime.min.js"></script>
	<title>Otaku Village | Accueil</title>
	<meta charset="utf-8">
</head>
<body>
	<div class="rectangle1">
		<div class="tiret1"></div>
		<nav class="menu-nav">
			<ul>
				<li class="btn">
					<a href="#">
						<i class="fas fa-home"></i>
						Accueil
					</a>
				</li>
				<li class="btn">
					<a href="dev.html">
						<i class="far fa-address-card"></i>
						Candidatures
					</a>
				</li>
				<li class="btn">
					<a href="dev.html">
						<i class="fas fa-feather"></i>
						Notre team
					</a>
				</li>
				<li class="btn">
					<a href="dev.html">
						<i class="fas fa-trophy"></i>
						Classement teams
					</a>
				</li>
				<li class="btn">
					<a href="dev.html">
						<i class="fas fa-info"></i>
						Informations
					</a>
				</li>
				<li class="btn">
					<a href="https://discord.gg/PSKjefq">
						<i class="fab fa-discord"></i>
						Notre discord
					</a>
				</li>
			</ul>
		</nav>
		<nav class="menu-connection">
			<a href="#" class="info" id="notif">
				<img class="cloche-1" src="img/cloche1.png" width="25" height="25">
				<span>Notifications</span>
			</a>
			<ul>
				<li class="btn-connection">
					<a href="index.php">
						<i class="fas fa-sign-in-alt"></i>
						Déconnection
					</a>
					<?php session_destroy(); ?>
				</li>
			</ul>
		</nav>
	</div>
	<h1 class="text-search">Rechercher un Pseudo (Exemple: Admin)</h1>
	<div class="search">
		<form action="/" method="post" class="search_layout">
			<input type="text" id="search" name="q" placeholder="Rechercher">
			<input type="submit" value>
			<label for="submit" name="formsend" id="formsend">
				<div class="search_btn">
					<i class="fas fa-search"></i>					
				</div>
			</label>
		</form>
	</div>
	<?php
		include 'bdd.php';
		global $db;

		$pseudo = $db->query('SELECT pseudo FROM users');

		if(isset($_POST['q']) AND !empty($_POST['q'])){
			extract($_POST);
			$q = htmlspecialchars($_POST['q']);
			$pseudo = $db->query('SELECT pseudo FROM users WHERE pseudo LIKE "%'.$q.'%"');
			header ("Locaton: profill.php?$q");
		}
	?>
	<?php if($pseudo->rowCount() > 0) { ?>
	<?php } else { ?>
		<?php header ('Location: erreurl.php'); ?>
	<?php } ?>
	<style>
		div.aucun-champ{
			position: absolute;
			top: 0px;
			left: 0px;
		}
	</style>
	<style type="text/css">
		div.search_btn{
			position: absolute;
			top: 11px;
			left: 223px;
			font-size: 18px; 
			color: rgba(136,136,136,1);
		}
		div.search{
			position: absolute;
			top: 200px;
			left: 900px;
		}
		.search input[type='submit'] {
			background-color: hsl(0, 0%, 23%); 
			border-style: solid; 
			border-width: 1px;
			height: 39px;
			border-radius: 3px;
			width: 46px; 
			border-color: hsl(0, 0%, 23%); 
			border: solid 1px hsl(0, 0%, 18%);
			position: absolute;
			top: 0px;
			left: 210px;
		}
		.search input[type='text'] {
			height: 25px;
			background: rgb(18, 18, 18);
			border: none;
			color: #D2D2D2;
			border: solid 1px rgba(40, 40, 40, .8);
			padding-left: 10px;
			outline: 0;
			font-size: 18px;
			line-height: 1.3333333;
			border-radius: 6px;
			padding: 6px 12px;
			border-radius: 0px;
			transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
		}		
		select, input, button, textarea, button {
			font: 99% arial,helvetica,clean,sans-serif;
		}
	</style>
	<h1 class="ml12">Bienvenue sur Otaku Village <i class="fas fa-child"></i> :</h1>
	<p class="text-bvn-back-p">Ici vous pouvez trouvez <br> des choses par rapport au serveur.</p>
	<a class="btn-fleche-droite2" href="#">
		<i class="fas fa-angle-left"></i>
	</a>
	<a class="btn-fleche-gauche2" href="#">
		<i class="fas fa-angle-right"></i>
	</a>
	<a class="savoirplus" href="ensavoirplus.html">En savoir plus <i class="fas fa-angle-double-right"></i></a>
	<div class="top-2">
		<img id="background-principal" src="img/background-principal.jpg" class="background-principal" width="1475" height="500" style="display: block;">
		<img id="background-principal" src="img/background-principal2.jpg" class="background-principal" width="1475" height="500" style="display: none;">
		<img id="background-principal" src="img/background-principal3.jpg" class="background-principal" width="1475" height="500" style="display: none;">
	</div>
	<div class="carre3">
		<img class="img-carre-2" src="img/bot.png" width="260" height="260">
		<h1 class="bottext">Bot</h1>
		<p class="bottextp">Un bot personnalisé et en développement...</p>
	</div>
	<div class="carre2">
		<img class="img-carre-2" src="img/japon.png" width="260" height="260">
		<h1 class="japontext">Japon</h1>
		<p class="japontextp">Sur notre serveur <a class="url" href="https://discord.gg/PSKjefq">discord</a> vous aurez tout se qui tourne autour du japon.</p>
	</div>
	<div class="carre">
		<img class="img-carre-1" src="img/securiter.png" width="200" height="200">
		<h1 class="secutext">Sécurité</h1>
		<p class="secutextp">Nous tentons de sécurisé le bot et le serveur pour évité le piratage par des personnes malveillantes.</p>
	</div>
	<div>
		<a id="cRetour" class="cInvisible" href="#haut">
			<i class="fas fa-angle-up"></i>
		</a>
	</div>
	<div class="footer-background">
		<p class="footer-text"><small>2018 Otaku Village™ | Aucun droits réserver</small></p>
	</div>
</body>
</html>
<!--Site fait par Vidon_ :p--!>
